Simplenote/BannerView.swift

```diff
@@ -55,17 +49,19 @@ class SustainerView: UIView {
     // MARK: - Actions
 
     @IBAction
-    func sustainerWasPresssed() {
+    func bannerWasPresssed() {
         onPress?()
     }
 }
 
 // MARK: - Private API(s)
 //
-private extension SustainerView {
+private extension BannerView {
 
-    func refreshInterface() {
-        let style = isActiveSustainer ? Style.sustainer : Style.notSubscriber
+    func refreshInterface(with style: Style? = nil) {
```

Comment:
Modified this function so that we can define a style for the BannerView instead of using the predefined sustainer Styles that exist in the file

Reply:
If this API doesn't do anything whenever `Style` is nil, perhaps we should drop the optional param?